tests/wicked/wlan/sut/t01_open_static_ip.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test WiFi setup with wicked (Open static IP)
+# - WiFi Access point:
+#   - Use virtual wlan devices
+#   - AP with hostapd is running in network namespace
+# - WiFi Station:
+#   - connect using ifcfg-wlan1 and `wicked ifup`
+#   - check if STA is associated to AP
+#   - ping both directions AP <-> STA
+#
+# Maintainer: cfamullaconrad@suse.com
+
+
+use Mojo::Base 'wicked::wlan';
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Setup ref
+    $self->netns_exec('ip addr add dev wlan0 ' . $self->ref_ip . '/24');
```

Comment:
why we setting up ref in sut ? it is messy ... can we move this to ref ?

Reply:
no, this hwsim only works on **one** host.